vmngclient/api/versions_utils.py

```diff
@@ -144,12 +145,30 @@ def get_device_list_in_available(self, version_to_set_up: str, devices: List[Dev
 
         Args:
             version_to_set_up (str): requested version
+            devices (List[Device]): devices on which action going to be performed
 
         Returns:
             list : list of devices
         """
         return self._get_device_list_in(version_to_set_up, devices, "available_versions")
 
+    def get_devices_current_version(self, devices: List[Device]) -> List[dict]:
```

Comment:
Looks like all methods in this module uses List[dict] but data is structured
Is it possible to use typed list?
eg. List[DeviceId]
where new dataclass is declared in this module:
```python
@dataclass
class DeviceId():
  deviceId: str
  deviceIP: str
```

Reply:
It's possible, but it's scope for the next PR.